packages/core/src/components/Modal/private/ImageLayout.tsx

```diff
@@ -61,47 +62,32 @@ export default withStyles(({ responsive }) => ({
   },
 
   splitContentImagePane: {
+    borderLeft: ui.border,
     alignItems: 'center',
     display: 'flex',
     justifyContent: 'center',
     overflow: 'hidden',
     position: 'relative',
 
     '@media': {
-      [responsive.small]: {
+      [`(max-width: ${breakpoints.xsmall}px)`]: {
```

Comment:
Should we just add `responsive.xsmall`?

Reply:
i'd be down for it! wasn't sure, so i took a safer route